corehq/apps/app_manager/views/download.py

```diff
@@ -365,12 +380,36 @@ def validate_form_for_build(request, domain, app_id, unique_form_id, ajax=True):
         return HttpResponse(response_html)
 
 
-def download_index_files(app):
+def _requested(path, prefix, profiles):
```

Comment:
Maybe a better function name or a docstring about what the function's intending to detect; it isn't obvious to me just looking at the code/diff